packages/components/src/components/link.css

```diff
@@ -16,6 +16,10 @@
 		text-decoration-thickness: 0.2em;
 	}
 
+	:is(a, button):is(:focus, :hover):is(:disabled, [aria-disabled], [disabled]) {
```

Comment:
Wir können alternativ auch den Selektor in L15 anpassen. Fände ich insgesamt leichter verständlich.

```
:is(a, button):is(:focus, :hover):not(:disabled, [aria-disabled]) {
```

Das `[disabled]` dürfte redundant sein, da bereits mit `:disabled` abgedeckt.